### PR TITLE
[TS] LPS-200609

### PR DESCRIPTION
**Pull request by MirnaGama**
### Context:
[LPS-200609 - Edit client credentials modal causes blank space above page and collapses screen](https://liferay.atlassian.net/browse/LPS-200609)

### Steps to reproduce:

1. Go to Control Panel > Security > OAuth 2 Administration
2. Select any "OAuth 2 Administration"
3. Click "Edit" button on "Client ID" or "Client Secret"

Actual result: An unnecessary blank space appears at the top of the screen.
Expected result: Unnecessary blank space is not displayed.

### Implementation details:
 The best long term solution would be to migrate away from new `A.Modal`. The amount of logic that’s being handled by javascript in the jsp could be handled much more simply if more of it is migrated to React. We would then be using the Clay Modal and not have to worry about things being as fragile in the future.

### Commits included:
- LPS-200609 Add react and clay libs
- LPS-200609 Client credentials modal component
- LPS-200609 Component for include the two modals
- LPS-200609 Replace the Modal component and functions with a new component created with React 
- LPS-200609 Remove unused liferay-portlet tag
- LPS-200609 Auto generated

### CODE REVIEW

**Comment by MirnaGama** [comment]
ci:test:sf

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_credentials.jsp [hunk 1]
```diff
@@ -164,70 +159,6 @@ String clientSecret = (oAuth2Application == null) ? "" : oAuth2Application.getCl
...
-
-				<div class="closed">
-					<clay:icon symbol="lock" /><liferay-ui:message key="unchanged" />
-				</div>
-			</div>
-
-			<aui:input helpMessage="client-secret-id" label="client-secret" name="newClientSecret" onKeyup='<%= liferayPortletResponse.getNamespace() + "updatePadlock('clientSecretPadlock', this.value, '" + HtmlUtil.escapeJS(clientSecret) + "')" %>' type="text" value="<%= clientSecret %>" />
-
-			<aui:button-row>
-				<aui:button href="" icon="icon-plus" onClick='<%= liferayPortletResponse.getNamespace() + "generateRandomSecret()" %>' value="generate-new-secret" />
```
Comment (gaborlovasdotcom) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_credentials.jsp:223]: This change removes the Generate New Secret button from the UI

**Comment by MirnaGama** [comment]
Hi @gaborlovasdotcom 

I had to reformulate this solution, the components were made with javascript and react because I needed to build a valid resource URL and the theme display attributes were difficult to receive in typescript. Now the client secret modal has both buttons working as expected.

/cc @oliverkecskemety

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:relevant

**Comment by MirnaGama** [comment]
ci:test:oidc

**Comment by MirnaGama** [comment]
ci:test:oauth2

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by oliverkecskemety** [comment]
ci:stop

**Pull request merged**